Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.cpp

```diff
@@ -364,16 +364,16 @@ namespace AzToolsFramework::ViewportUi::Internal
     {
         // no background for the widget else each set of buttons/textfields/etc would have a black box around them
         SetTransparentBackground(widget);
-        widget->setWindowFlags(Qt::Window | Qt::FramelessWindowHint);
+        widget->setWindowFlags(Qt::Window | Qt::FramelessWindowHint | Qt::WindowDoesNotAcceptFocus);
     }
 
     void ViewportUiDisplay::InitializeUiOverlay()
     {
-        m_uiMainWindow.setObjectName(m_uiMainWindow.windowTitle());
+        m_uiMainWindow.setObjectName(QString("ViewportUiWindow"));
```

Comment:
Nit I missed before: we can use `QStringLiteral` here to avoid a copy in the QString constructor.
It might also be nice to add a comment stating that this impacts focus behavior

Reply:
Ah sorry I missed this too, as this only happens one time it likely isn't too bit of an issue in terms of the copy, though good to know about `QStringLiteral` for future use. I'm on the fence about actually adding a comment here as this was broken behavior from before (`windowTitle` was empty) and it is related to some rather irritating code in `PatchedAction` that in future might wind-up being removed anyway. If I do come back to this I'll definitely see if I can add additional context (and have my sights on a test soon too!)